src/components/ZeroLogon/Zerologon.tsx

```diff
@@ -94,31 +135,54 @@ export function ZeroLogon() {
         setAllowSave(true);
     };
 
+    /**
+     * Clears the output state.
+     */
     const clearOutput = useCallback(() => {
         setOutput("");
-        setAllowSave(false);
         setHasSaved(false);
-    }, [setOutput]);
+        setAllowSave(false);
+    }, []);
 
+    /**
+     * Handles the completion of saving the output to a text file.
+     */
     const handleSaveComplete = useCallback(() => {
         setHasSaved(true);
         setAllowSave(false);
     }, []);
 
     return (
-        <form onSubmit={form.onSubmit((values) => onSubmit({ ...values }))}>
-            {LoadingOverlayAndCancelButton(loading, pid)}
-            <Stack>
-                {UserGuide(title, description_userguide)}
-                <TextInput label={"DC Name"} required {...form.getInputProps("dcName")} />
-                <TextInput label={"Target IP"} required {...form.getInputProps("targetIP")} />
-                <TextInput label={"Administrative Name@DC-IP"} required {...form.getInputProps("adminName")} />
-                <TextInput label={"Hashes"} required {...form.getInputProps("hashes")} />
-
-                <Button type={"submit"}>Exploit</Button>
-                {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
-                <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
-            </Stack>
-        </form>
+        <RenderComponent
+            title={title}
+            description={description}
+            steps={steps}
+            tutorial={tutorial}
+            sourceLink={sourceLink}
+        >
+            {!loadingModal && (
+                <InstallationModal
+                    isOpen={opened}
+                    setOpened={setOpened}
+                    feature_description={description}
+                    dependencies={dependencies}
+                />
+            )}
+            <form onSubmit={form.onSubmit(onSubmit)}>
+                <Stack>
+                    {LoadingOverlayAndCancelButton(loading, pid)}
+                    <TextInput label={"DC Name"} required {...form.getInputProps("dcName")} />
```

Comment:
Change label to "Domain controller (DC) name" in case somebody has not read the user guide. 
```suggestion
                    <TextInput label={"Domain controller (DC) name"} required {...form.getInputProps("dcName")} />
```